ui/app/components/console/ui-panel.js

```diff
@@ -8,22 +9,38 @@ import {
   executeUICommand,
 } from 'vault/lib/console-helpers';
 
-const { inject, computed } = Ember;
+const { inject, computed, getOwner, run } = Ember;
 
 export default Ember.Component.extend({
   classNames: 'console-ui-panel-scroller',
   classNameBindings: ['isFullscreen:fullscreen'],
   isFullscreen: false,
   console: inject.service(),
+  router: inject.service(),
   inputValue: null,
   log: computed.alias('console.log'),
 
+  didReceiveAttrs() {
+    let val = this.get('inputValue');
```

Comment:
I don't think it's necessary for the `didXyz` component hooks, but I try to always call `this._super(...arguments)` whenever overriding a hook. That way if you have a mixin or a reopened class using the same hook, you don't accidentally break things.

Reply:
Just got rid of the in favor of always running it in didRender.